databroker/tests/test_broker.py

```diff
@@ -197,6 +197,30 @@ def test_uid_lookup():
     assert_raises(ValueError, lambda: db[uid[0]])
 
 
+def test_find_by_float_time():
+    uid = insert_run_start(time=100., scan_id=1,
+                           owner='nedbrainard', beamline_id='example',
+                           uid=str(uuid.uuid4()))
+    result = db(start_time=99, stop_time=101)
+    uids = [hdr['start']['uid'] for hdr in result]
+    assert uid in uids
+
+def test_find_by_string_time():
+    uid = insert_run_start(time=ttime.time(), scan_id=1,
+                           owner='nedbrainard', beamline_id='example',
+                           uid=str(uuid.uuid4()))
+    today = datetime.today()
+    yesterday = '{}-{}-{}'.format(today.year,
+                                  str(today.month).zfill(2),
+                                  str(today.day-1).zfill(2))
```

Comment:
This will only fail on the first of the month.  There is a `timedelta` in `datetime` that knows how to do date arithmetic.